<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addData</class>
 <widget class="QDialog" name="addData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление новых данных</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resourse.qrc">
    <normaloff>:/photo/add.ico</normaloff>:/photo/add.ico</iconset>
  </property>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>401</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="resource">
        <property name="text">
         <string>Ресурс</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="adResource"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="login">
        <property name="text">
         <string>Логин</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="addLogin"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="password">
        <property name="text">
         <string>Пароль</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="addPassword"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resourse.qrc"/>
 </resources>
 <connections/>
</ui>
